moveit_core/collision_detection_fcl/src/collision_common.cpp

```diff
@@ -39,6 +39,7 @@
 #include <fcl/shape/geometric_shapes.h>
 #include <fcl/octree.h>
 #include <boost/thread/mutex.hpp>
+#include <boost/weak_ptr.hpp>
```

Comment:
I guess it worked fine before due to transitive includes. It's used for example in line 343.